website/docs/dev_colorspace.md

```diff
@@ -0,0 +1,113 @@
+---
+id: dev_colorspace
+title: Colorspace Management and Distribution
+sidebar_label: Colorspace
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+Defines the distribution of colors and OCIO config during publishing. Once colorspace data are captured and integrated into representation loaders could use them for loading image and video data with correct colorspace.
+
+:::warning Color Management (ImageIO)
+Settings schema is requred for any host or module which is processing pixel data.
+:::
+
+## Data model
+The *colorspaceData* are stored at root of representation dictionary during publishing. Once they are integrated into representation db document they are stored as *representation_doc.data["colorspaceData"]*
```

Comment:
```suggestion
Published representations that are extracted with color managed data store a **colorspaceData** entry in its data: `representation_doc["data"]["colorspaceData"]`.

It's up to the Host implementation to pre-configure the application or workfile to have the correct OCIO config applied.
It's up to the Extractors to set these values for the representation during publishing.
It's up tot he Loaders to read these values and apply the correct expected color space.
```

Reply:
Typo `It's up tot he Loaders ` -> `It's up to the Loaders `

Reply:
Fixed the typo in the suggestion. Thanks.